docs/book/advanced-guide/pipelines/scheduling-runs.md

```diff
@@ -0,0 +1,114 @@
+[Vertex Orchestrator](../../component-gallery/orchestrators/gcloud-vertexai.md) only supports the cron expression method (see below).
+
+You could write a cron expression to describe the pipeline schedule in terms
+that would be comprehensible as [a cron job](https://en.wikipedia.org/wiki/Cron). For example, if you wanted your pipeline to run at 14:05 on Wednesdays, you could use the following:
+
+```python
+from zenml.config.schedule import Schedule
+from zenml.pipelines import pipeline
+
+pipeline_instance = pipeline(step_one=..., step_two=...)
+
+schedule = Schedule(cron_expression="5 14 * * 3")
+pipeline_instance.run(schedule=schedule)
+```
+
+Alternatively, you could manually specify start and end times and the interval between runs. For example, if you wanted your pipeline to run once every 30 minutes starting right now, you could use the following:
+
+```python
+from zenml.config.schedule import Schedule
+from zenml.pipelines import pipeline
+
+pipeline_instance = pipeline(step_one=..., step_two=...)
+
+schedule = Schedule(start_time=datetime.now(), interval_second=1800)
+pipeline_instance.run(schedule=schedule)
+```
+
+You can also optionally specify an `end_time` for the schedule if you want it not
+to run after a certain moment. Similarly, the boolean `catchup` parameter can be
+used to specify whether a recurring run should catch up (i.e. backfill pipeline
+runs) if it is behind schedule, if you paused the schedule, for example.
+
+In the context of scheduled cron or pipeline jobs, backfilling refers to running
+a missed job for a specific period in the past. For example, if a pipeline
+misses a scheduled run at 12:00 PM, backfilling can be used to run the pipeline
+for the 12:00 PM time slot, in order to collect the missing data. This helps
+ensure that the pipeline is up-to-date and that downstream jobs have the
+necessary data to run correctly. Backfilling is a useful technique for catching
+up on missed runs and filling in gaps in scheduled jobs, and can help ensure
+that pipelines and cron schedules are running smoothly. Usually, if your
+pipeline handles backfill internally, you should turn catchup off to avoid
+duplicate backfill. (Here's [a handy
+guide](https://medium.com/nerd-for-tech/airflow-catchup-backfill-demystified-355def1b6f92)
+in the context of Airflow.) Note that the `catchup` parameter enabling
+backfilling is not supported in all orchestrators.
+
+# How to stop or pause a scheduled run
+
+The way pipelines are scheduled depends on the orchestrator you are using. For example, if you are using Kubeflow, you can use the Kubeflow UI to stop or pause a scheduled run. If you are using Airflow, you can use the Airflow UI to do the same. However, the exact steps for stopping or pausing a scheduled run may vary depending on the orchestrator you are using. We recommend consulting the documentation for your orchestrator to learn the current method for stopping or pausing a scheduled run.
```

Comment:
Ok you should add here that currently ZenML only gets involved to initially schedule a job, and to maintain the lifecycle of the schedule is still on the user (with the methods you described).

Maybe explain what happens if you run a scheduled pipeline twice. @schustmi actually what happens in thsi case? Does it just create two pipelines with different names?